receiver/mysqlreceiver/templates/querySample.tmpl

```diff
@@ -0,0 +1,41 @@
+SELECT
+    thread.thread_id,
+    COALESCE(thread.processlist_user, ''),
+    COALESCE(processlist.host, ''),
+    COALESCE(thread.processlist_db, ''),
```

Comment:
Please also fetch and send
thread.processlist_command
thread.processlist_thread